src/firefly/java/edu/caltech/ipac/firefly/server/security/OidcAdapter.java

```diff
@@ -307,4 +333,30 @@ private boolean validateClaims(JWTClaimsSet claimsSet) {
                 && claimsSet.getExpirationTime().getTime() > System.currentTimeMillis()
                 && claimsSet.getIssueTime().getTime() < System.currentTimeMillis();
     }
+
+    @NotNull
+    private static RoleList getRoleList(List<String> isMemberOf) {
+        RoleList roleList = new RoleList();
+        for (String r : isMemberOf) {
+            String cn=null, ou=null, dc=null;
+            for(String s : r.split(",")) {
+                String[] kv = s.split("=");
+                if (kv.length > 1) {
+                    if (kv[0].equalsIgnoreCase("cn")) {
+                        cn = cn == null ? kv[1] : cn;
+                    } else if (kv[0].equalsIgnoreCase("ou")) {
+                        ou = ou == null ? kv[1] : ou;
+                    } else if (kv[0].equalsIgnoreCase("dc")) {
```

Comment:
For ignorant people like me it would be helpful to know what 'ou', 'cn', and 'dc' are...
May be adding a link to some docs somewhere in OidcAdapter.javawould help.